sdk/messaging/azservicebus/migrationguide.md

```diff
@@ -43,6 +34,10 @@ New (using `azservicebus`):
 // new code
 
 client, err := azservicebus.NewClientFromConnectionString(connectionString, nil)
+
+//  or, if you would like to use an `azidentity` credential:
```

Comment:
```suggestion
//  or, if you would like to use an azidentity credential:
```
But also consider splitting this into its own block so you can show more context, like importing `azidentity` and calling `NewDefaultAzureCredential`. That might help people getting started with track 2.

Reply:
Updated. I had a section later on in the doc specifically dedicated to the use of Azure Identity credentials, so I just linked to that.